dtfs-central-api/src/utils/string.ts

```diff
@@ -0,0 +1,5 @@
+export const isString = (value: unknown): value is string => typeof value === 'string' || value instanceof String;
+
+export const isEmptyString = (str: unknown) => !str || (isString(str) && !str.trim().length);
+
+export const hasValue = (str: unknown) => str && !isEmptyString(str);
```

Comment:
What do we mean by `hasValue`?

Reply:
Looks like it means defined and not empty. This is just refactored to TS from existing code, so isn't something I'd want to spend too long refactoring further now.